### PR TITLE
DUI: Curve was considered as namespace on top of revit

### PR DESCRIPTION
**Pull request by aosyatnik**
I've added GEOMETRY_CURVE_ACTION, so that it can be used in Revit project.
[PR in RevitProject](https://github.com/DynamoDS/DynamoRevit/pull/194).

Reviewers
@Benglin , please take a look.

Link to YouTrack:
[MAGN-6528](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6528) DUI: Curve was considered as namespace on top of revit

### CODE REVIEW

**Comment by aosyatnik** [comment]
As we decided in [this PR](https://github.com/DynamoDS/DynamoRevit/pull/194). The best place to store "Spacing Rule Layout" is Revit>Elements>DividedPath>Action.
@Benglin , please, take a look.

File: src/DynamoCore/Nodes/NodeCategories.cs [hunk 1]
```diff
@@ -92,6 +92,7 @@ public static class BuiltinNodeCategories
         public const string REVIT_PARAMETERS = "Revit.Parameters";
         public const string REVIT_BAKE = "Revit.Bake";
         public const string REVIT_API = "Revit.API";
+        public const string REVIT_ELEMENTS_DIVIDEDPATH_ACTION = "Revit.Elements.DividedPath.Actions";
```
Comment (Benglin) [inline src/DynamoCore/Nodes/NodeCategories.cs]: Hi @aosyatnik, Revit specific things have no place in `DynamoCore`, please move all these Revit stuff out of this repository, and add them to `DynamoDS/DynamoRevit:Revit2014` branch (then later create two other PRs for both `Revit2015` and `Revit2016` branches).

While you're doing this, can you also remove those strings that are no longer used? Like when I casually searched, I see that `LOGIC_MATH_ROUNDING` is not used anywhere. Only a few under `Core` is used. Thanks!

Comment (aosyatnik) [inline src/DynamoCore/Nodes/NodeCategories.cs]: Are you sure about deleting `LOGIC_MATH_ROUNDING`. Even if it's not used in `DynamoCore`, it doesn't mean it's not used somewhere else.
E.g. `DynamoRevit` use some strings from BuiltinNodeCategories, but `DynamoCore` doesn't use it. I'm afraid if I delete something, something from other connected project can be broken.

Comment (aosyatnik) [inline src/DynamoCore/Nodes/NodeCategories.cs]: @Benglin , I've changed my [Revit PR](https://github.com/DynamoDS/DynamoRevit/pull/194). But in my opinion it's better to use string from BuiltinNodeCategories(as it was before), than use this hardcoded string. But I couldn't find any other place to store it.

Comment (Benglin) [inline src/DynamoCore/Nodes/NodeCategories.cs]: Hi @aosyatnik, I'm so sorry for the delay, it's been a very tight week for me. As mentioned above I'd want those unused stuff out of `NodeCategories.cs`. Whatever that is being used in Revit, should be moved into a new `NodeCategories.cs` file in `DynamoDS/DynamoRevit` repository. See comment I left in [your Revit PR](https://github.com/DynamoDS/DynamoRevit/pull/194).

**Comment by aosyatnik** [comment]
Hi, @Benglin , I removed all unused categories. If you merge it, please ping me, so that I'll be near, if something breaks. I tried run it on Revit 2014, it worked fine.

**Comment by Benglin** [comment]
Thanks a lot for going through the items one by one, @aosyatnik. I appreciate the efforts! LGTM.

**Pull request merged**